### ISSUE TITLE
Req: yLineMinValueHeight

### ISSUE THREAD

**Issue by skuske** [Zirkfied/ZFChart#7]
Beside yLineMaxValueHeight, there should also be a yLineMinValueHeight for charts that have values not below a specific value, for example. :)

**Comment by BiggerGao** [comment]
我也觉得，可以分个mode出来，做成DefaultMode(现在这种)和DoubleExtendMode

**Comment by Zirkfied** [comment]
@skuske   You mean there is a distance between  0 to the xAxisLine?

**Comment by skuske** [comment]
No, there should be an option to let me specifiy at which value the Y-achsis should start. For example: I want to display the atmospheric pressure in a bar chart. For such a chart it makes no sense to let the Y-achsis start at 0 as the atmosperic pressure never goes below somewhere at 920mb... So, in such a case it would be cool to make the y-achsis start at a value of 900 or something and not at 0. :)

**Comment by skuske** [comment]
Maybe it should be called 'minYachsisValue'

**Comment by Zirkfied** [comment]
OK. I see, I will add this function in next version ! Thanks for your advice

**Comment by skuske** [comment]
Brilliant. Keep up the good work! :)

**Comment by skuske** [comment]
I just throught that you could also make a boolean option, and if set to 'true', that option would use the lowest value from the array of values as the minimum Y-achsis value. Let's say you have an array of 4 values: 910, 980, 940 and 1002. The y-achsis should then use the lowest value (910) or a slightly lower value as the minimum for the y-achsis.

**Comment by Zirkfied** [comment]
It has been updated

**Comment by skuske** [comment]
:) Thanks!
